frontend/src/__tests__/cypress/cypress/support/commands/odh.ts

```diff
@@ -315,6 +328,147 @@ declare global {
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/pipelines/:type`,
+        options: { path: { namespace: string; type: string }; times?: number },
+        response: OdhResponse<PipelineKFv2>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/runs`,
+        options: { path: { namespace: string }; times?: number },
+        response: OdhResponse<PipelineRunKFv2>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/runs`,
+        options: { path: { namespace: string } },
+        response: OdhResponse<ListPipelineRunsResourceKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/runs/:runId`,
+        options: { path: { namespace: string; runId: string } },
+        response: OdhResponse<GenericStaticResponse<string, Body>>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/experiments`,
+        options: { path: { namespace: string } },
+        response: OdhResponse<ListExperimentsResponseKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/experiments/:experimentId`,
+        options: { path: { namespace: string; experimentId: string } },
+        response: OdhResponse<GenericStaticResponse<string, Body>>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/experiments/:experimentId`,
+        options: { path: { namespace: string; experimentId: string } },
+        response: OdhResponse<ExperimentKFv2>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/recurringruns`,
+        options: { path: { namespace: string }; times?: number },
+        response: OdhResponse<PipelineRunJobKFv2>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/recurringruns`,
+        options: { path: { namespace: string } },
+        response: OdhResponse<ListPipelineRunJobsResourceKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `DELETE /api/service/pipelines/:namespace/dspa/apis/v2beta1/runs/:runId`,
+        options: { path: { namespace: string; runId: string } },
+        response: OdhResponse<K8sStatus>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/runs/:runId`,
+        options: { path: { namespace: string; runId: string } },
+        response: OdhResponse<PipelineRunKFv2 | { error: GoogleRpcStatusKF } | undefined>,
```

Comment:
`{ error: GoogleRpcStatusKF }` is not a valid response type.
The corresponding code currently sets this type of response needs to be investigated and fixed.

Reply:
Just because the test is written this way doesn't mean it's correct. Please look at the frontend code to see what it expects and align with the kubeflow api.